eng/pipelines/templates/steps/typespec-ci.yml

```diff
@@ -14,7 +14,14 @@ steps:
     pushd ${{parameters.Folder}}
 
     # Pass "--no" to ensure npx does not install anything (TypeSpec compiler should already be installed)
-    npx --no tsp compile . --warn-as-error || exit_code=1
+    if test -f tspconfig.yaml; then
+      if test -f main.tsp; then
+        npx --no tsp compile . --warn-as-error || exit_code=1
+      fi
+      if test -f main.tsp; then
+        npx --no tsp compile client.tsp --no-emit --warn-as-error || exit_code=1
+      fi
+    fi
```

Comment:
```suggestion
```

Reply:
I think i resolved this, lmk if not let me know